zmon-controller-app/src/main/resources/config/application.yml

```diff
@@ -50,6 +50,12 @@ zmon:
       checkid: 9
     eventlog:
         url: http://localhost:8081/
+        connectTimeout: 30000
+        requestConnectTimeout: 30000
```

Comment:
The connection timeouts can default to much lower values. I suggest we define 5000ms for both `connectTimeout` and `requestConnectTimeout`